docs/reference/run.md

```diff
@@ -509,6 +509,7 @@ container:
 |--------------------------------------|---------------------------------------------------------------------------------------------|
 | `-m`, `--memory="" `                 | Memory limit (format: , where unit = b, k, m or g)                                          |
 | `--memory-swap=""`                   | Total memory limit (memory + swap, format: , where unit = b, k, m or g)                     |
+| `--kernel-memory=""`                 | Kernel memory limit (format: <number><optional unit>, where unit = b, k, m or g)            |
```

Comment:
Can you add ticks around `<number><optional unit>`? Otherwise they won't show properly in HTML. Perhaps we should use `<number>[<unit>]` to mark that "unit" is optional.

Looks like that "format" got lost in the lines above it as well (not introduced by this PR)